src/wallet/rpcwallet.cpp

```diff
@@ -5012,7 +5012,7 @@ UniValue z_sendmany(const UniValue& params, bool fHelp)
     int nMinDepth = parseMinconf(DEFAULT_NOTE_CONFIRMATIONS, params, 2, std::nullopt);
 
     std::optional<CAmount> nFee;
-    if (params.size() > 3 && params[3].get_real() != -1.0) {
+    if (params.size() > 3 && !params[3].isNull()) {
```

Comment:
How do we specify `null` when calling this via `zcash-cli`? Is it the unquoted string `null`?

Reply:
I believe so.

Reply:
I'll try this out locally.

Reply:
Yep – it’s already working and testable in other operations (like `setban`).